readme.md

```diff
@@ -51,6 +51,7 @@
 - [npm-top](https://gist.github.com/bcoe/dcc961b869bbf6685002) - npm users by downloads.
 - [npm semver calculator](http://semver.npmjs.com) - Visually explore what versions of a package a semver range matches.
 - [npm-stats](http://www.npm-stats.com) - Displays metrics about packages.
+- [my-npm-stats](https://my-npm-stats.org) - Package stats and dependents for NPM authors.
```

Comment:
I think the description could be more specific. That it's weekly downloads. And it's `npm`.

Reply:
@sindresorhus I have two ideas on this:
1. Check/track the success of your NPM packages. (use case centered)
2. Weekly downloads and dependent counts for NPM package authors. (feature centered, this might be extended in the future)

If you have an idea from the outside, I'm very open to it 😊.

Reply:
You're asking the wrong person as I'm not your target audience.